examples/usb_serial.rs

```diff
@@ -0,0 +1,66 @@
+//! CDC-ACM serial port example using polling in a busy loop.
+#![no_std]
+#![no_main]
+
+extern crate panic_semihosting;
+
+use cortex_m_rt::entry;
+use stm32l0xx_hal::usb::{Peripheral, UsbBus};
+use stm32l0xx_hal::{pac, prelude::*, rcc};
+use usb_device::prelude::*;
+use usbd_serial::{SerialPort, USB_CLASS_CDC};
+
+#[entry]
+fn main() -> ! {
+    let dp = pac::Peripherals::take().unwrap();
+
+    let mut rcc = dp.RCC.freeze(rcc::Config::hsi16());
```

Comment:
The example doesn't set up the HSI48 clock:
```suggestion
    let mut rcc = dp.RCC.freeze(rcc::Config::hsi16());
    let mut syscfg = SYSCFG::new(dp.SYSCFG_COMP, &mut rcc);
    rcc.enable_hsi48(&mut syscfg, dp.CRS);
```

The old API enforced this by requiring a `HSI48` as an argument. I think it's unfortunate that the new API doesn't do this, but I think this can be improved upon later.